docs/community/09_grants.md

```diff
@@ -0,0 +1,30 @@
+---
+id: grant-program
+title: Grants
+---
+
+# Radworks Grants Program
+The Radworks Grants program is used to support open source initives that support the development of the Radworks ecosystem. Specifically, the Grants program focuses on funding:  
+- Additions to core Radicle software or components of Web3 infrastructure that interact with the Radicle stack
+- Additions to core Drips software or components, or compelling adoption of tooling, such as Radicle Drips SDK
+- Initiatives furthering the adoption of Radicle & Drips
+
+Most grants distributed via the Grants program fall between XX-XX USDC/RAD. All grant applications are reviewed by the Grants Committee - a group of trusted ecosystem participants. Grants are distributed using Drips on a milestone-based schedule.
```

Comment:
A few thoughts on the range of amount to put here:
- Based on current conversations around Grants restructuring, we're likely going to move towards only supporting retroactive grants
- Our smallest retroactive grant was ~2,000 USDC, which funded some very early work from Jason before he joined Drips full time
- We probably want to leave some room for larger projects/ideas. The largest grants we made were upwards of 100,000 USDC (e.g. Yorgos). However, we likely won't have space for this size if we're transitioning to retroactive only, as it's unlikely someone would work for "free" on months of work with the chance of possibly being retroactively funded later on down the line. So I think topping it off around 50,000 USDC, which would be 2 highly skilled developers working full time for 2-3 months.

So I think a range we could start with is:

2,000-50,000 USDC

Reply:
Got it, thanks for the context. I will add this in